test/likelihoods/test_softmax_likelihood.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import unittest
+import torch
+from gpytorch.likelihoods import SoftmaxLikelihood
+from torch.distributions import Distribution
+from gpytorch.distributions import MultivariateNormal
+from test.likelihoods._base_likelihood_test_case import BaseLikelihoodTestCase
+
+
+class TestSoftmaxLikelihood(BaseLikelihoodTestCase, unittest.TestCase):
+    seed = 0
+
+    def _create_conditional_input(self, batch_shape=torch.Size([])):
+        return torch.randn(*batch_shape, 6, 5)
+
+    def _create_marginal_input(self, batch_shape=torch.Size([])):
+        mat = torch.randn(*batch_shape, 6, 5, 5)
+        return MultivariateNormal(torch.randn(*batch_shape, 6, 5), mat @ mat.transpose(-1, -2))
+
+    def _create_targets(self, batch_shape=torch.Size([])):
+        return torch.distributions.Categorical(
+            probs=torch.tensor([0.25, 0.25, 0.25, 0.25])
+        ).sample(torch.Size([*batch_shape, 5]))
+
+    def create_likelihood(self):
+        return SoftmaxLikelihood(num_features=6, num_classes=4)
+
+    def _test_conditional(self, batch_shape):
+        likelihood = self.create_likelihood()
+        input = self._create_conditional_input(batch_shape)
+        output = likelihood(input)
+
+        self.assertTrue(isinstance(output, Distribution))
```

Comment:
`self.assertIsInstance` - also in other places

Reply:
```suggestion
        self.assertIsInstance(output, Distribution)
```